src/Components/Book/Books.js

```diff
@@ -25,20 +26,46 @@ const Bookss = ({ item_id, title, author }) => {
       <ul>
         {books.map((item) => (
           <div key={item.item_id} className="book-list">
+            <div className="title-comment">
             <DisplayBooks
+              category={item.category}
               title={item.title}
               author={item.author}
             />
-            <button type="button" id={item.item_id} onClick={(e) => { handleDelete(e); }}>Remove</button>
+            <div className='btn-container'>
+              <button type="button" className="comment-btn">Comment</button>
+              <button type="button" className="remove-btn" id={item.item_id} onClick={(e) => { handleDelete(e); }}>Remove</button>
+              <button type="button" className='edit-btn'>Edit</button>
+            </div>
+            </div>
+
+            <div className="status-container">
+            <dt className="prog-report">
+            <p><FaCircleNotch size={100} style={{ color: '#0290ff' }} /></p>
+              <div className="status">
+                <dl className="percent">85%</dl>
+                <dl className="complete">Completed</dl>
+              </div>
+            </dt>
+
+            <dt className="chapter-container">
+                <dl className="chapter">CURRENT CHAPTER</dl>
+                <dl className="topic">Chapter 8</dl>
+              <button className="prog-btn" type="button">UPDATE PROGRESS</button>
+            </dt>
+          </div>
           </div>
         ))}
       </ul>
-      <div>
+      {/* <div>
         {title}
       </div>
       <div>
         {author}
       </div>
+      <div>
+        {item_id}
+      </div> */}
```

Comment:
[OPTIONAL]:
- [ ] To maintain a clean codebase, kindly remove unused pieces of code. Please do not forget to have a look at the other files to do as suggested.